### PR TITLE
REPO-210: added optional param isFavorite.

### PR DESCRIPTION
**Pull request by anechifor**
added optional isFavorite using parameter INCLUDE_ISFAVORITE.

### CODE REVIEW

File: src/main/java/org/alfresco/rest/api/impl/NodesImpl.java [hunk 1]
```diff
@@ -914,6 +913,12 @@ else if (type.equals(Type.FOLDER))
             boolean isLocked = isLocked(nodeRef, aspects);
             node.setIsLocked(isLocked);
         }
+
```
Comment (cristianturlica) [inline src/main/java/org/alfresco/rest/api/impl/NodesImpl.java]: Code formatting needed

File: src/main/java/org/alfresco/rest/api/impl/NodesImpl.java [hunk 2]
```diff
@@ -914,6 +913,12 @@ else if (type.equals(Type.FOLDER))
             boolean isLocked = isLocked(nodeRef, aspects);
             node.setIsLocked(isLocked);
         }
+        
+        if(includeParam.contains(PARAM_INCLUDE_ISFAVORITE))
```
Comment (cristianturlica) [inline src/main/java/org/alfresco/rest/api/impl/NodesImpl.java]: Code formatting needed.

**Comment by dcaruana** [comment]
Looks good :)

**Comment by montgolfiere** [comment]
Looks fine.

Have you also "documented" within the API spec ? For example, at least as part of each method where this can be used as well as maybe noted within the "create favourite" method ?

**Pull request merged**